apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.sql

```diff
@@ -1,23 +1,21 @@
 -- BLOCK select_job
 SELECT
-  CASE
-    WHEN ai.id IS NULL THEN NULL
-    ELSE to_jsonb(
-      authz_assessment_instance (
-        ai.id,
-        $authz_data,
-        $req_date,
-        ci.display_timezone,
-        a.group_work
-      )
-    )
-  END AS aai,
```

Comment:
Testing reveals that you need to update the query params that are passed to this.

<img width="4202" height="1838" alt="Image" src="https://github.com/user-attachments/assets/c1d976b7-a979-4fff-8f95-c9595baf4708" />